assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -37,38 +61,42 @@ spec:
-        - --secure-port=6443
-        - --service-account-key-file=/etc/kubernetes/secrets/service-account.pub
-        - --service-cluster-ip-range={{ .Values.apiserver.serviceCIDR }}
-        - --storage-backend=etcd3
-        - --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt
-        - --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key
-        {{- if .Values.apiserver.enableAggregation }}
-        - --proxy-client-cert-file=/etc/kubernetes/secrets/aggregation-client.crt
-        - --proxy-client-key-file=/etc/kubernetes/secrets/aggregation-client.key
-        - --requestheader-client-ca-file=/etc/kubernetes/secrets/aggregation-ca.crt
-        - --requestheader-extra-headers-prefix=X-Remote-Extra-
-        - --requestheader-group-headers=X-Remote-Group
-        - --requestheader-username-headers=X-Remote-User
-        {{- end }}
+        - /bin/sh
+        - -c
+        - |
+          set -x && \
+          exec /hyperkube \
+          kube-apiserver \
+          --advertise-address=$(POD_IP) \
+          --allow-privileged=true \
+          --anonymous-auth=false \
+          --authorization-mode=RBAC \
+          --bind-address=$(cat /run/kube-apiserver/address) \
+          --client-ca-file=/etc/kubernetes/secrets/ca.crt \
+          --cloud-provider={{ .Values.apiserver.cloudProvider }} \
+          --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,Priority,PodSecurityPolicy \
+          --etcd-cafile=/etc/kubernetes/secrets/etcd-client-ca.crt \
+          --etcd-certfile=/etc/kubernetes/secrets/etcd-client.crt \
+          --etcd-keyfile=/etc/kubernetes/secrets/etcd-client.key \
+          --etcd-servers={{ .Values.apiserver.etcdServers}} \
+          --insecure-port=0 \
+          --kubelet-client-certificate=/etc/kubernetes/secrets/apiserver.crt \
+          --kubelet-client-key=/etc/kubernetes/secrets/apiserver.key \
+          --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname \
+          --secure-port={{ template "port" . }} \
+          --service-account-key-file=/etc/kubernetes/secrets/service-account.pub \
+          --service-cluster-ip-range={{ .Values.apiserver.serviceCIDR }} \
+          --storage-backend=etcd3 \
+          --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt \
+          {{ if .Values.apiserver.enableAggregation -}}
+          --proxy-client-cert-file=/etc/kubernetes/secrets/aggregation-client.crt \
+          --proxy-client-key-file=/etc/kubernetes/secrets/aggregation-client.key \
+          --requestheader-client-ca-file=/etc/kubernetes/secrets/aggregation-ca.crt \
+          --requestheader-extra-headers-prefix=X-Remote-Extra- \
+          --requestheader-group-headers=X-Remote-Group \
+          --requestheader-username-headers=X-Remote-User \
+          {{ end -}}
+          --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key
```

Comment:
Should this be next to `--tls-cert-file=`?

Reply:
This is used to make handling last backslash easier. But agreed, it would be nice to have it paired. I'll move some other parameter to the bottom.

Reply:
Done. PTAL.